src/constants.ts

```diff
@@ -0,0 +1,32 @@
+export const MIN_SCANNER_VERSION_REQUIRED = '3.20.0'
+
+export const INTERNAL_OUTFILE = 'SalesforceCodeAnalyzerResults.json'
+
+/* eslint-disable prettier/prettier, prefer-template */
+export const MESSAGES = {
+    STEP_LABELS: {
+        PREPARING_ENVIRONMENT: 'Preparing Environment',
+        RUNNING_CODE_ANALYZER: 'Running the Salesforce Code Analyzer',
+        UPLOADING_ARTIFACT: 'Uploading Artifact',
+        ANALYZING_RESULTS: 'Analyzing Results',
+        FINALIZING_OUTPUT: 'Finalizing Summary and Outputs'
+    },
+    MISSING_NORMALIZE_SEVERITY: `Missing required --normalize-severity option from run-arguments input.`,
+    SF_CLI_NOT_INSTALLED:
+        `The sf command was not found.\n` +
```

Comment:
The tick character (\`) is used to declare multiline strings with wildcards.
So you could just do:
```
`The sf command was not found.
The Salesforce CLI must be installed blah blah`
```
And that would be fine, no `\n` character needed.

Reply:
The issue is that it picks up all of the whitespace if I do as you suggested and I didn't want to break the linter. So this is what is recommended actually.